TF_Tooling_Web/src/Cursors/SvgCursor.ts

```diff
@@ -89,7 +89,7 @@ export class SVGCursor extends TouchlessCursor {
 
         const position = _inputAction.CursorPosition;
 
-        if (position) {
+        if (position && this.shouldShow) {
```

Comment:
I could be worth just having `shouldShow` as an early return before doing anything with this method instead. We don't really need to be doing anything with the cursor if it shouldn't be there!

Reply:
Good shout, added!